@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/core/fn_coreGetWCAs.sqf

```diff
@@ -71,7 +74,14 @@ if (_eng1PwrLvrState == "FLY" || _eng2PwrLvrState == "FLY") then {
 if (getpos _heli # 2 >= 3) then {
 	_onGnd = false;
 };
-private _rtrRPM = ((_heli animationPhase "mainrotorRPM") * 1.08) / 10;
+private _rtrRPM = 0.0; 
+if (_flightModel == "SFMPlus") then {
```

Comment:
SFMPlus, the core sim, uses the animation phase for rotor RPM because it's 100% SFM which does it's own wierd things. The SimpleRotor side uses the actual output Np from the engine sim, which will eventually be dynamic. SFMPlus will always use the animationPhase and the simpleRotor will always have it's own discrete simulation.